src/sentry/apidocs/hooks.py

```diff
@@ -162,6 +193,18 @@ def custom_preprocessing_hook(endpoints: Any) -> Any:  # TODO: organize method,
                 ApiPublishStatus.EXPERIMENTAL: set(),
             }
 
+        # Fail if endpoint doesn't implement pagination correctly
+        if method == "GET":
+            endsWithObject = re.search(r"/([^/{}]+)s/$", path)
+            if endsWithObject is not None:
+                class_name = callback.view_class.__name__
+                file_path = inspect.getfile(callback.view_class)
```

Comment:
`inspect.getfile` is quite slow.  I would maybe recommend just using `__module__` and then `__file__` directly